transports/janus_mqtt.c

```diff
@@ -956,32 +1254,32 @@ int janus_mqtt_client_subscribe(janus_mqtt_context *ctx, gboolean admin) {
 	MQTTAsync_responseOptions options = MQTTAsync_responseOptions_initializer;
 	options.context = ctx;
 	if(admin) {
-#ifdef MQTTVERSION_5
+	#ifdef MQTTVERSION_5
 		if(ctx->connect.mqtt_version == MQTTVERSION_5) {
 			options.onSuccess5 = janus_mqtt_client_admin_subscribe_success5;
 			options.onFailure5 = janus_mqtt_client_admin_subscribe_failure5;
 		} else {
 			options.onSuccess = janus_mqtt_client_admin_subscribe_success;
 			options.onFailure = janus_mqtt_client_admin_subscribe_failure;
 		}
-#else
+	#else
 		options.onSuccess = janus_mqtt_client_admin_subscribe_success;
 		options.onFailure = janus_mqtt_client_admin_subscribe_failure;
-#endif
+	#endif
```

Comment:
See above on macros indentation (not going to write it for them all, as I see there are others: please fix them all).